frontend/src/components/doctor/AddNewDoctor.tsx

```diff
@@ -1,19 +1,38 @@
 import DoctorForm from "./DoctorForm";
 import { Heading } from "@chakra-ui/react";
 import { useNavigate } from "react-router-dom";
-import { DoctorDetail } from "./DetailDoctorPage";
+import { Doctor } from "./DoctorControl"; // type import
```

Comment:
You can specify that you're importing a type like this:

```suggestion
import type { Doctor } from "./DoctorControl";
```